app/controllers/movies_controller.rb

```diff
@@ -19,4 +17,23 @@ def index
       @movies << JSON.parse(page2.body, symbolize_names: true)[:results]
     end
   end
+
+  def show
+    movie_response = conn.get("/3/movie/#{params[:id]}?api_key=#{ENV['MOVIE_API_KEY']}&language=en-US")
+    @movie = JSON.parse(movie_response.body, symbolize_names: true)
+
+    review_response = conn.get "/3/movie/#{params[:id]}/reviews?api_key=#{ENV['MOVIE_API_KEY']}&language=en-US&page=1"
+    reviews = JSON.parse(review_response.body, symbolize_names: true)
+    @reviews = reviews[:results]
+
+    cast_response = conn.get "/3/movie/#{params[:id]}/credits?api_key=#{ENV['MOVIE_API_KEY']}"
+    credits = JSON.parse(cast_response.body, symbolize_names: true)
+    @cast = credits[:cast]
+  end
+end
+
+private
+
+def conn
+  Faraday.new('https://api.themoviedb.org')
 end
```

Comment:
This method doesn't need to be public and is available to all methods within the class. Thanks for implementing this.